core/domain/prod_validation_jobs_one_off.py

```diff
@@ -316,39 +380,56 @@ def _get_external_model_properties(cls):
         This should be implemented by subclasses.
 
         Returns:
-            tuple(str, list(tuple), dict). A tuple with first element as
-            external model name, second element as a tuple of
-            cls.external_instance_details and the third element
-            as a properties dict with key as property name in summary
-            model and value as property name in external model.
+            tuple(str, str, dict). A tuple with first element as
+            external model name, second element as a key to fetch
+            external model details from
+            cls.field_name_to_external_model_references
+            and the third element as a properties dict with key as
+            property name in summary model and value as property name
+            in external model.
 
         Raises:
             NotImplementedError. This function has not yet been implemented.
         """
         raise NotImplementedError
 
     @classmethod
-    def _validate_external_model_properties(cls, item):
+    def _validate_external_model_properties(
+            cls, item, field_name_to_external_model_references):
         """Validate that properties of the model match the corresponding
         properties of the external model.
 
         Args:
             item: ndb.Model. BaseSummaryModel to validate.
+            field_name_to_external_model_references:
+                dict(str, (list(ExternalModelReference))).
+                A dict is keyed by field name. Each value consists of a list
```

Comment:
This docstring just describes the structure of the dict, but does not explain what it represents.

Reply:
Updated.